inspirehep/utils/helpers.py

```diff
@@ -42,23 +40,6 @@ def download_file(url, output_file=None, chunk_size=1024):
     return output_file
 
 
-def download_file_to_workflow(workflow, name, url):
```

Comment:
This really didn't belong to `utils.helpers`: I have moved it into `modules.workflows.utils` where it rather belong.